test/Benchmarks/src/Vector/Map_Error_Benchmark.enso

```diff
@@ -22,11 +22,11 @@ id i = i
 id_one_error i = if i != 5 then i else
     Error.throw (Illegal_Argument.Error "error")
 
-always_throw i =
+id_always_throw i =
     _ = [i]
     Error.throw (Illegal_Argument.Error "error")
 
-throw_ten_percent i = if i > 0 then i else
+id_throw_ten_percent i = if i > 0 then i else
```

Comment:
The function throws for zero and we have numbers 0..9 - that's very predictable and one could easily lead to special casing in the implementation. Anyway, and this is bigger:

### TCR - Technical Change Required

How is this function different from 
```ruby
id_one_error i = if i != 5 then i else
    Error.throw (Illegal_Argument.Error "error")
```
they **both** select a single digit and throw on it. E.g. both cases are measuring _10%_ in my opinion!

Reply:
Has this been addressed by some commit?